includes/classes/CatalogBuilder.php

```diff
@@ -215,11 +212,10 @@ public function set_post_block_terms( $post_id, $output ) {
 	/**
 	 * Builds a list of Block Term names for a given post.
 	 *
-	 * @param int   $post_id The post id.
-	 * @param array $opts Optional args
+	 * @param int $post_id The post id.
 	 * @return array
 	 */
-	public function get_post_block_terms( $post_id, $opts = [] ) {
+	public function get_post_block_terms( $post_id ) {
```

Comment:
@dsawardekar Is there any reason to remove this param? If you are refactoring, then can you check compatibility? I notice that few functions are passing `$opt`.

<img width="913" alt="image" src="https://github.com/10up/block-catalog/assets/1784821/e9f21d97-b422-4eb5-91eb-080463f69f44">

Reply:
@ravinderk I've added back the extra opts.